exasol/toolbox/tools/replace_version.py

```diff
@@ -0,0 +1,16 @@
+from pathlib import Path
+from typing import List
+
+def replace_version(template: Path, version: str) -> None:
+    with open(template, "r+", encoding="utf8") as file:
+        lines = file.readlines()
+        _replace(lines, "exasol/python-toolbox/.github/", version)
+        file.seek(0)
+        file.writelines(lines)
+
+
+def _replace(lines: List[str], replace_filter: str, version: str) -> None:
```

Comment:
Usually you want to have the functionality handle a single item, multiple then be handled by adding a loop and potentially wrapping.

e.g.:

```python
def _replace(line, filter, version):
     ....

def _replace_line(lines, filter, version):
      for l in lines:
          _replace(line, filter, version)
```

Additionally you could use "one" function and distinguish using [single_dispatch](https://docs.python.org/3/library/functools.html#functools.singledispatch)
`_replace(...)`, `replace(...)`